src/main/java/net/pms/network/UPNPHelper.java

```diff
@@ -217,12 +217,14 @@ public static void sendAlive() {
 		LOGGER.debug("Sending ALIVE...");
 		MulticastSocket multicastSocket = null;
 		SocketAddress sa = null;
-		NetworkInterface ni = null;
+		NetworkInterface ni = PMS.get().getServer().getNetworkInterface();
+		try {
+			sa = new InetSocketAddress(getIPv4MulticastAddress(), UPNP_PORT);
+		} catch (IOException e1) {
```

Comment:
Is this an error we want to ignore? What are the consequences of this error if it happens?

Reply:
@SubJunk better?